src/app/ui/MusicSelectScene.jsx

```diff
@@ -283,53 +288,53 @@ export const MusicSelectScene = React.createClass({
       Analytics.send('MusicSelectScene', 'select chart')
       this.props.onSelectChart(this.props.musicSelect.song, chart)
     }
-  },
-  handleFilter (e) {
+  }
+  handleFilter = (e) => {
     this.props.onFilterTextChange(e.target.value)
-  },
-  handleOptionsOpen () {
+  }
+  handleOptionsOpen = () => {
     Analytics.send('MusicSelectScene', 'open options')
     this.setState({ optionsVisible: true })
-  },
-  handleOptionsClose () {
+  }
+  handleOptionsClose = () => {
     this.setState({ optionsVisible: false })
-  },
-  handleCustomBMSOpen () {
+  }
+  handleCustomBMSOpen = () => {
     this.setState({ customBMSVisible: true })
     Analytics.send('MusicSelectScene', 'open custom BMS')
-  },
-  handleCustomBMSClose () {
+  }
+  handleCustomBMSClose = () => {
     this.setState({ customBMSVisible: false })
-  },
-  handleCustomSong (song) {
+  }
+  handleCustomSong = (song) => {
     this.setState({ customBMSVisible: false })
-  },
-  handleUnofficialClick () {
+  }
+  handleUnofficialClick = () => {
     this.setState({ unofficialDisclaimerVisible: true })
     Analytics.send('MusicSelectScene', 'view unofficial disclaimer')
-  },
-  handleUnofficialClose () {
+  }
+  handleUnofficialClose = () => {
     this.setState({ unofficialDisclaimerVisible: false })
-  },
-  handleLogout () {
+  }
+  handleLogout = () => {
     if (confirm('Do you really want to log out?')) {
       Promise.resolve(online.logOut()).done()
       Analytics.send('MusicSelectScene', 'logout')
     }
-  },
-  handleAuthenticate () {
+  }
+  handleAuthenticatev = () => {
```

Comment:
I think this is misspelt — it broke the log-in/sign-up dialog.

Reply:
Ah, whoops, totally missed that!